Neovici/cosmoz-omnitable
# wip

Testing instructions:

1. open http://localhost:8000/demo/lit-omnitable.html
you will see a 20000 item omnitable
2. scroll the window
3. sort the entire dataset by clicking on the column headers
4. switch sort direction by clicking same header twice
